frontend/packages/metal3-plugin/src/components/baremetal-hosts/BareMetalHostDetails.tsx

```diff
@@ -127,13 +130,18 @@ const BareMetalHostDetails: React.FC<BareMetalHostDetailsProps> = ({
             <dd>
               <BareMetalHostStatus {...status} />
             </dd>
-            <dt>Power Status</dt>
-            <dd>
-              <StatusIconAndText
-                title={powerStatus}
-                icon={<BareMetalHostPowerStatusIcon powerStatus={powerStatus} />}
-              />
-            </dd>
+            {/* power status is not available until host registration/inspection is finished */}
+            {!HOST_REGISTERING_STATES.includes(provisioningState) && (
```

Comment:
Wouldn't it be better to say "Not available"?

Reply:
I am following the convention used in details view of not showing items which are not populated. But maybe it makes sense to show `Not Available` in this case? @andybraren, comments?